node/views/index.ejs

```diff
@@ -275,12 +302,7 @@
         // payment token to be generated before the Link handler can be
         // initialized.
         handler = Plaid.create({
-          apiVersion: 'v2',
-          clientName: 'Plaid Quickstart',
-          env: '<%= PLAID_ENV %>',
-          product: products,
-          key: '<%= PLAID_PUBLIC_KEY %>',
-          countryCodes: '<%= PLAID_COUNTRY_CODES %>'.split(','),
+          ...linkHandlerCommonOptions,
```

Comment:
I might have asked this before, but why are we making a request to `get_access_token` here if it isn't relevant?

Reply:
The access token and item ID are displayed on the quickstart success page so this request it just to ensure the quickstart success pane still looks good.